.env.production.example

```
@@ -0,0 +1,72 @@
+# Authentication
+# ===================================
+# JWT_SECRET MUST be a strong random key (min 32 characters)
+# Generate with: openssl rand -base64 32
+JWT_SECRET=CHANGE-THIS-TO-STRONG-RANDOM-KEY-MIN-32-CHARS
+AUTH_SESSION_DURATION=7d
+
+# ===================================
+# Email (SES)
+# ===================================
+SES_FROM_EMAIL=no-reply@finance-manager.com
+SES_FROM_NAME=Finance Manager
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)